expect/mod.ts

```diff
@@ -48,25 +48,29 @@
  * - `toThrowErrorMatchingSnapShot`
  * - `toThrowErrorMatchingInlineSnapShot`
  *
- * The following asymmetric matchers are not supported yet:
- * - `expect.anything`
- * - `expect.any`
- * - `expect.arrayContaining`
- * - `expect.not.arrayContaining`
- * - `expect.closedTo`
- * - `expect.objectContaining`
- * - `expect.not.objectContaining`
- * - `expect.stringContaining`
- * - `expect.not.stringContaining`
- * - `expect.stringMatching`
- * - `expect.not.stringMatching`
+ * The asymmetric matchers are not yet fully supported:
+ * - Supported:
+ *   - `expect.anything`
+ *   - `expect.any`
+ *   - `expect.arrayContaining`
+ *   - `expect.not.arrayContaining`
+ *   - `expect.closedTo`
+ *   - `expect.stringContaining`
+ *   - `expect.not.stringContaining`
+ *   - `expect.stringMatching`
+ *   - `expect.not.stringMatching`
+ * - Not supported:
+ *   - `expect.objectContaining`
+ *   - `expect.not.objectContaining`
```

Comment:
This doesn't make sense. Within a list of symbols that are not fully supported, there's a list of symbols that are supported. Can you please correct and clarify?